src/runners/test_queries.py

```diff
@@ -0,0 +1,96 @@
+TEST_1_ALERT = """
+CREATE OR REPLACE VIEW rules.test1_alert_query COPY GRANTS
+  COMMENT = 'Test 1 Alert Query; should group with Test 3
+  @tags test-tag'
+AS
+SELECT OBJECT_CONSTRUCT('account', 'account_test', 'cloud', 'cloud_test') AS ENVIRONMENT
+    , ARRAY_CONSTRUCT('obj1', 'obj2') AS OBJECT
+    , 'test1_alert_query' AS TITLE
+    , 'This is a test alert query; this should be grouped with Test 3' AS DESCRIPTION
+    , 'SnowAlert' AS DETECTOR
+    , 'test_actor' AS ACTOR
+    , 'test action 1' AS ACTION
+    , 'test_1_query' AS QUERY_ID
+    , 'TEST1_ALERT_QUERY' AS QUERY_NAME
+    , 'low' AS SEVERITY
+    , ARRAY_CONSTRUCT('source') AS SOURCES
+    , OBJECT_CONSTRUCT('data', 'test data') AS EVENT_DATA
+    , current_timestamp() as EVENT_TIME
+    , current_timestamp() as ALERT_TIME
+FROM (select 1 as test_data)
+WHERE 1=1
+    AND test_data=1
+"""
+
+TEST_1_ALERT_GRANT = 'GRANT SELECT ON rules.test1_alert_query to role snowalert_testing'
```

Comment:
merging this would make more work for the schema-based grants that I'm working on, so let's go with finishing that first, still